### DISCUSSION TITLE
Resolve/include yaml file as ref?

### DISCUSSION THREAD

**Discussion by hardik-id** [OpenAPITools/openapi-diff#476] [General]
Hello, our openapi.yaml has a reference to other YAML files. 

So now when I use diff on two openapi.yaml file. would it resolve (include YAML file?) and then check the difference?

![image](https://user-images.githubusercontent.com/6074721/206147564-a8b1d819-527b-433f-9607-7cb8291fa9ce.png)

![image](https://user-images.githubusercontent.com/6074721/206147776-4ca4cdba-c49c-494e-a678-0477d61de73f.png)

**Comment by joschi** [comment]
@hardik-id In general this should be possible, but there are certainly some bugs.

Could you provide a minimal before/after OpenAPI specification for reproducing the issue?
